notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb

```diff
@@ -0,0 +1,378 @@
vI80jK8P6wicCvtDGDP/O8g9VZPyK+/A6ymcZ33SPoH4F+Aj5NWvZpNepK6Gx913ANiZmZmZmaV8RwQMzMzMzOrjBsgZmZmZmZWGTdAzMzMzMysMm6AmJmZmZlZZdwAMTMzMzOzyrgBYmZmZmZmlXEDxMzMzMzMKuMGiJmZmZmZVcYNEDMzMzMzq4wbIGZmXUDSnZJmDnYcnSbpYkkbmsz7QUk9knbvdFxmZlYdN0DMzEqQNClfHDfaLi1xyGh7kN0pgJ5igqTPSTqml7xD8nuR9LeSzpM0arBjMTPrNooYkud2M7NBJWkS8F/AOcAjdbsXRMS8AR7vDmBdRBzRngi7k6RhwPCI2FBIWwd8JyJOqcsrYEREPFtxmC2TdBZwKTAmIpYOdjxmZt1km8EOwMxsiLslIuYMdhBDRUT0UNcD0kfeALqi8SFp+4h4eiBFOhaMmdkQ5yFYZmYdJOlkST+T9ISkdZIWSPpQk2VPl7RQ0lpJKyXNlvSuujx/KWmKpOWS1kuan3tn+jv28Dxc7N8kvU/SAzm+2ZLe2CD/6yXdKunPklZLuk3SQXV5tpF0gaTF+VhPSvofSYcX8myeA1KLAXgRUJvv0SPpqrx/izkgkm6W9EAvn+ceSXfVpU2SdK+kpyWtkHRtM/NJcow9kvaRNFXSU8Av8r4DJH1b0pL8fS+TNFnSSwrlLyL1fgA8no+1qfjeZWMzM3s+cA+ImVlrRkvapZgQESsKLz8K3A/cCGwEjgW+KYmImNzbQSV9FLgC+H7+uR3wOuAQYHrOsyswm9RL8FVgBXAU8C1JO0TE15qI/++B9+TyG4DTgFsljY+IB/L7vA6YBawkXVj3AB8BZkk6tNADdAlwBvAN4D5gNHAQMI58AU9hXkdEbJL0XuBbwJ3A1TnPQ/V5s6nA1ZIOiIjfFL6rVwKvB04vpJ0HnAt8D5gM/EXef7CkcRGxpo/vpPaeNwC/A84q7Hs7MCbHuhx4LfBhYF/g0JxnGvAaYCLwMWBVTl/ZhtjMzIa+iPDmzZs3bwPcgEmkC/H6bVNdvpENyt4GLKpLuwOYWXj9Y2BOPzFMAR4FRtelTwP+SJo/0VvZ4TnejcBrC+mvANYD19XFspY0n6GWtjuwGritkDYfuKGfmC8Cnq1LWwdc1SDvycAmYPf8enSO7dK6fGfnz7Fbfv2q/Pozdfn2JzWyzmgixh5gSoN9jerzpBznIYW0s4qxF9Jbis2bN2/eng+bh2CZmZUXpB6Otxa2t22RIeKZ2r8lvTj3lswC9pG0XR/HXgXsKWlco515gvYEUs/KNpJ2qW3ATOAlwF838RnuiIgFhXj/l9TgODK/zzb5c82IiMcK+ZYC1wFvLnyOVcD+kl7dxPsOWET8ifTZJtbtmgjcGRHL8usTSHUzo+57WQYsAQ6nf0HqyamPoVifI/Nxf02a83FgE8dtR2xmZkOah2CZmbXmnuhjErqkw4ALgIOB7Qu7gnRHf10vRb8AvAW4T9Ji0oX3tRFxd96/K7AjcCpp2FS9IA3t6c9DDdIeBI6XtDNp6NfInFZvEaknZQ9gMWlFsBuAxZLmAzeTVrda2EQczZoKXJOHiN0raSxwAOl7qHlNjmtJg/IB/LnJ9wBiH2sAAAPuSURBVPp9fUJuLJxPavS8rO64o5s4ZrtiMzMbstwAMTPrEEl7k4ZbLQA+BTxGmq/xTuDj9LEQSET8Nl9cv4PUG3ECcJqkcyLikkLZbwPf7eUwv+klvSMi4vbc+3EscATwIeAzkk6OiGva9DY3As+QGgD35p8bgRmFPMNy2pG9HGN1k+/VqHE4gzTf5DJgHmlo2gjgJppb2KVdsZmZDVlugJiZdc47SRenR0fEE7VESW9vpnCkZV+nAdMkjSBdfJ8j6YukCdBrgWER8fMWYty7QdpYYHVErMxDsJ7JafX2Jc1zeLwQ81OkuSlTJO0A/JLUY9BXA6TpB1JFxBpJN5EaHmfmn7dHxJOFbA+Texki4pFmj92f3PvxJuDsiLiskP5XjULt5TAdic3MbCjxHBAzs87ZlH9uPtfm5Vrf31/BPPxps0gP7vtdPtaIiNgE/ACYKGnfBuVf2mSMh+ZVrmrl9iL1utyS33cjqRfneEl7FPLtBpxIuvhf10vMa0kX3CP7iWEtsFOT8UIahjVGaTnj/UhzUYpmkBoA5zUqXB/nAGxVn9mn2LrBsTb/rP9cnYrNzGzIcA+ImVl5/T1s7lbSUJ2bJE0GXkwalrSM/udn/FzSo8CvgCdIF9qnAjdGxPqc50zSHfnZ+fiLgJ2B8cBhpHki/VkIzJR0JWlo0Kn55wWFPJ8jTY6+S9LXSBfQHybdyS8uUfugpNtIS/A+RVoy+FjSMsJ9uQ84QtInSd/NwxFxbx/5fwI8DfwraeWoHxR3RsTivNTthXlI2I+ANaQVqCYAV5KWHR6QiFiVnzVydp54v5Q0lGpPtv5duC+nfUHS9BznDzsVm5nZUOIGiJlZeX0OHYqIRZL+EbiYdLG8lHSBuQb4Zj/H+zrwbtLd9VGkYU6X89wD7oiI5UoPAzwPOJ7U4FhBmnNSbBj05WfAHNJzKfYgLaX77ohYVHif+ZLeRJoY/9mcfDcwMSLuLxzrCuAY0vyPkcAjpCVyL+/jcwJ8krTi1MWkSe9Xk+Z3NBQR6yT9hDT86uY87Ks+zyWSFuVjn5uTHwN+SmrAlHUiqYHwsfw5bgGOJtXP5s8VEXfnhsYppGezDCM9P2RpB2MzMxsSFNH00FszM3uekDScdFf+yxHx6cGOx8zMXjg8B8TMzMzMzCrjBoiZmZmZmVXGDRAzsxeuYABL4JqZmbWD54CYmZmZmVll3ANiZmZmZmaVcQPEzMzMzMwq4waImZmZmZlVxg0QMzMzMzOrjBsgZmZmZmZWGTdAzMzMzMysMm6AmJmZmZlZZdwAMTMzMzOzyrgBYmZmZmZmlXEDxMzMzMzMKuMGiJmZmZmZVeb/Aerpa1m22UhkAAAAAElFTkSuQmCC"
```

Comment:
Please consider removing output cells from all notebooks.

Reply:
Thanks for your suggestion. I will remove the output cells. For the ROC curve, may I keep it as what we do in this example https://github.com/Azure/mmlspark/blob/master/notebooks/samples/Regression%20-%20Vowpal%20Wabbit%20vs.%20LightGBM%20vs.%20Linear%20Regressor.ipynb ?

Reply:
Hi Sudarshan, I've removed the output cells per your suggestion. Please let me know if it's ready to be merged to master.

Reply:
Thank you!